inferno/callbacks.py

```diff
@@ -267,6 +292,10 @@ class PrintLog(Callback):
     `BestLoss` callback takes care of creating those entries, which is
     why `PrintLog` works best in conjunction with that callback.
 
+    *Note*: `PrintLog` will not result in good outputs if the number
```

Comment:
Fair point. It is a minor issue at best so feel free to ignore. Pretty sure that people will understand what is meant.